src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler/DeprecationNotice.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Symfony\Bridge\PhpUnit\DeprecationErrorHandler;
+
+final class DeprecationNotice
+{
+    /**
+     * @var int
+     */
+    private $count = 0;
+
+    /**
+     * @var array<string,int>
+     */
+    private $countsByCaller = [];
+
+    public function addObjectOccurence($class, $method)
```

Comment:
```suggestion
    public function addMethodOccurence($class, $method)
```

? Dunno, but just "object" is misleading.

Reply:
I wanted to oppose that to procedural code. In procedural code, deprecations can happen outside functions, so I didn't use "functions", and I think the opposite of procedural would be object (or maybe object oriented? sounds a bit long)

Reply:
Not sure why are we talking about procedural code here? This function doesn't handle procedural code. Did you misplace your comment and it's a reply to some other thing? Yes this is OOP related concept, same as `method`, which I generally understand as function in class. I wanted to highlight with that that this method handles function deprecations of function within the class, not class itself, am I wrong?

Reply:
The thing is, deprecations from procedural code are not necessarily from functions, see for instance https://github.com/symfony/symfony/blob/af4035d4ecf98df3d9dec9ac9eaadd6ca6d00873/src/Symfony/Bridge/PhpUnit/Tests/DeprecationErrorHandler/weak.phpt#L20

So while you're certainly not wrong, using "method" would create a conceptual asymmetry between "method" vs "procedural code". "object" vs "procedural code" seems more balanced to me.

Reply:
Cool that's what needed to be clarrified. Then name is fine :+1